test/studies/shootout/mandelbrot/ldelaney/mandelbrot-unrolled-DynamicNDIter.chpl

```diff
@@ -0,0 +1,65 @@
+use DynamicIters;
+config const n = 200,
+             maxIter = 49,
+             limit = 4,
+             chunkSize = 1;
+
+proc main() {
+  var invN = 2.0 / n;
+  var Crb, Cib : [0..(n+6)] real;
+  const ydim = 0..n-1;
+  const xdim = 0..((n+7)/8)-1;
+  var pixels : [ydim][xdim] uint(8);
+
+  forall i in ydim {
+    Cib[i] = i * invN - 1.0;
+    Crb[i] = i * invN - 1.5;
+  }
+
+  forall xy in dynamic({xdim, ydim}, chunkSize, 0, 2){
```

Comment:
You don't need to change this, but in case you weren't aware you can write a forall like this to do the tuple expansion for you:

```chpl
forall (x,y) in dynamic(...)
```